assets/utils.tsx

```diff
@@ -59,6 +59,10 @@ const getCoverageDateTimeFormat = () =>
 const getCoverageDateFormat = () => getLocaleFormat('COVERAGE_DATE_FORMAT');
 
 export const TIME_FORMAT = getTimeFormat();
+export const TIME_FORMAT_24_HOUR_CLOCK: boolean = new Intl.DateTimeFormat(
```

Comment:
imo you should rather check for `HH:` in TIME_FORMAT, otherwise this is not using NH settings really

Reply:
> otherwise this is not using NH settings really

It is using NH settings, but I realized we're using locale, and not time format directly which might cause problems if you wanted different time format from locale. Nikola will update it to use the time format